### PR TITLE
Fix the generated unique tag for Java and Python multi-arch containers

### PR DESCRIPTION
**Pull request by celeste-zeng**
-Added another step to generate the unique tag as the current timestamp.
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml [hunk 1]
```diff
@@ -99,6 +97,8 @@ jobs:
           PY_VER=${{ matrix.python_version }}
           PY_VER_CLEAN=${PY_VER//.}
           echo "py_ver_clean=$PY_VER_CLEAN" >> $GITHUB_OUTPUT
+      - name: Generate MULTIARCH_TAG unique variable based on timestamp
+        run: echo MULTIARCH_TAG=$(date +'%Y%m%d-%H%M%S%N') >> $GITHUB_ENV
```
Comment (tvalentyn) [inline .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:101]: how does this work?

Comment (tvalentyn) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:101]: is this a GH-action-y way of adding environment variable?

Comment (tvalentyn) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:101]: is $GITHUB_ENV a file?

Comment (celeste-zeng) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:101]: reference: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-setting-an-output-parameter

**Review by tvalentyn — APPROVED** [review]

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml (hunk 1 above)
Comment (tvalentyn) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml]: thanks!

**Pull request merged**